src/smif/data_layer/datafile_interface.py

```diff
@@ -493,7 +493,9 @@ def _set_list_coords(self, list_):
             self._set_item_coords(item)
 
     def _set_item_coords(self, item):
-        if 'dims' in item:
+        """If dims exists and is not empty
+        """
+        if 'dims' in item and item['dims']:
```

Comment:
Do we need to handle the `else` case here? In general, we should be clear about what's an error, and what's possible to skip and get away with by relying on defaults.

Reply:
Agreed